src/Type/Php/FilterVarDynamicReturnTypeExtension.php

```diff
@@ -91,6 +97,24 @@ private function getFilterTypeMap(): array
 		return $this->filterTypeMap;
 	}
 
+	/**
+	 * @return array<int, list<string>>
+	 */
+	private function getFilterTypeOptions(): array
+	{
+		if ($this->filterTypeOptions !== null) {
+			return $this->filterTypeOptions;
+		}
+
+		$this->filterTypeOptions = [
+			$this->getConstant('FILTER_VALIDATE_INT') => ['min_range', 'max_range'],
+			// PHPStan does not yet support FloatRangeType
+			// $this->getConstant('FILTER_VALIDATE_FLOAT') => ['min_range', 'max_range'],
```

Comment:
I don't plan to use it immediately, but it is possible to make `FloatRangeType` with `-INF <= float <= INF`.